packages/rnv/src/setupTools/base.js

```diff
@@ -147,6 +150,12 @@ class BasePlatformSetup {
         return true;
     }
 
+    async installAws() {
+        // to be overwritten
+        logError('Install aws not supported yet. Follow https://docs.aws.amazon.com/cli/latest/userguide/install-macos.html to install it manually (version 1 required)');
```

Comment:
this is the base class for platform setup. Each OS extends this, including `darwin`. If you add this error here it will be present on all platforms (`win`, `linux`, `darwin`) and you're referring to something related to `darwin`. For `darwin` you can follow the brew examples present in the `darwin` class for installing stuff with brew.